UI/src/app/app.module.ts

```diff
@@ -63,7 +67,20 @@ const appRoutes: Routes = [
     SharedModule,
     LoginModule,
   ],
-  providers: [],
+  providers: [{
+    provide: "BASE_API_URL",
+    useValue: 'http://localhost:3004'
```

Comment:
Consider keeping the value in `environment.ts`. To be able to set it depending on env.

Reply:
done